src/DashboardService.Models.Dto/Requests/Group/PatchGroupRequest.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace LT.DigitalOffice.DashboardService.Models.Dto.Requests.Group;
+
+public record PatchGroupRequest
+{
+  public Guid BoardId { get; set; }
+  public string GroupName { get; set; }
+  public bool GroupIsActive { get; set; }
```

Comment:
this is only for remove request - no need this property here

Reply:
we can repair this object use Patch

Reply:
Ivan is right actually)